### PR TITLE
database connection, config, driver code

### PR DESCRIPTION
**Pull request by megdye**

### CODE REVIEW

File: Olympick_project/Public API/all_codes/all_functions.py [hunk 1]
```diff
@@ -101,21 +103,85 @@ def find_event(sport_name):
...
+    elif operator == '2':
+        array_remove = remove_event(result)
+        remove_event_from_database(username, array_remove)
+        operator = input("Would you like to\n 1. Add more events\n 2. See your schedule again\n 3. Would you like to quit?")
+        if operator == '1':
+            add_events(username, password)
+        elif operator == '2':
+            get_entire_schedule(username)
+        else:
+            quit()
```
Comment (melanieet) [inline Olympick_project/Public API/all_codes/all_functions.py:160]: We might need to add a *try/catch block here incase someone decides to be awkward and input a number that isn't 1 or 2! What do you think?

*Try/except. Try/catch is JavaScript 🤭

File: Olympick_project/Public API/all_codes/all_functions.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 # show the location of the events
 import requests
 from datetime import datetime, timedelta
+from db_utils import verify_password, verify_existing_username, verify_new_username, remove_event_from_database, get_entire_schedule, add_event_to_database
```
Comment (melanieet) [inline Olympick_project/Public API/all_codes/all_functions.py:7]: I'm not sure we need to import every function, unless that's not every function? Maybe `import db_utils` would work just as well here!

**Review by melanieet — APPROVED** [review]
This is such great work, @megdye ✨. I'm beyond impressed! I noticed a couple of tiny things, but nothing that really needs to be addressed right now. And as you mentioned, we probably need to clean up the codebase before submitting anyway!

**Review by FaiqaDevel0per — APPROVED** [review]

**Comment by FaiqaDevel0per** [comment]
This is very good! Excellent work, although I might need you to explain me some of it 😬

**Pull request merged**